pkgs/applications/emulators/dosbox-staging/default.nix

```diff
@@ -1,39 +1,42 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, SDL2
-, SDL2_net
-, alsa-lib
+{ alsa-lib
```

Comment:
That was not the idea...

lib, stdenv and the fetchers go first; the others go in alphabetical order.